domains/Rbfl.localplayer.dev.json

```diff
@@ -0,0 +1,18 @@
+{
+    "description": "The official website of the team RBFL",
+    "domain": ".localplayer.dev",
+    "subdomain": "RBFL",
+
+    "owner": {
+        "repo": "https://github.com/WasiTC/Rbfl",
+        "email": "wasihossain692@gmail.com"
+    },
+
+    "record": {
+        "A": ["59.152.96.161"],
+        "TXT": ["google-site-verification=gWggOMxPBF8R9hohDmlgbvq5yH8Yh_jGD5mtxAM28oY"],
+        "NS": ["josephine.ns.cloudflare.com", "toby.ns.cloudflare.com"]
```

Comment:
We no longer accept NS records as of September 4th, 2024. You will not be able to add them to your subdomain.

Reply:
There i removed ns records